XmlSchemaClassGenerator.Console/Program.cs

```diff
@@ -90,15 +90,32 @@ static void Main(string[] args)
                 { "cc|complexTypesForCollections", "generate complex types for collections (default is true)", v => generateComplexTypesForCollections = v != null },
             };
 
-            var files = options.Parse(args);
+            var globsAndUris = options.Parse(args);
 
             if (showHelp)
             {
                 ShowHelp(options);
                 return;
             }
 
-            files = files.SelectMany(f => Glob.ExpandNames(f)).Concat(files.Where(f => Uri.IsWellFormedUriString(f, UriKind.Absolute))).ToList();
+            var uris = new List<string>();
+            foreach (var globOrUri in globsAndUris)
+            {
+                if (Uri.IsWellFormedUriString(globOrUri, UriKind.Absolute))
+                {
+                    uris.Add(globOrUri);
+                    continue;
+                }
+
+                var expandedGlob = Glob.ExpandNames(globOrUri).ToList();
+                if (expandedGlob.Count == 0)
+                {
+                    System.Console.WriteLine($"No matches for glob {globOrUri}");

```

Comment:
The error message should be more user-friendly: "No files found for ..." or similar.